### ISSUE TITLE
Erron when save imported css file in dev (hotAPI.install is not a function)

### ISSUE THREAD

**Issue by AlexVFornazieri** [SimulatedGREG/electron-vue#395]
##### Describe the issue / bug.
webpack-internal:///205:25 Uncaught TypeError: hotAPI.install is not a function
    at eval (webpack-internal:///205:25)
    at eval (webpack-internal:///205:36)
    at Object.<anonymous> (renderer.js:1968)
    at __webpack_require__ (renderer.js:678)
    at fn (renderer.js:89)
    at eval (webpack-internal:///204:3)
    at Object.<anonymous> (renderer.js:1962)
    at __webpack_require__ (renderer.js:678)
    at fn (renderer.js:89)
    at eval (webpack-internal:///203:4)

##### How can I reproduce this problem?

1. Importing any css project file in renderer main.js;
2. Then 'npm run dev' in terminal;
3. And with it running edit and save the imported css file

**The result here are:**
The main window turn white and error above is displayed in console log

I do not know if there is another way, maybe something more correct to import a global style file, but as frameworks' css imports are made like this, I believe it is correct, and it works normally, but like describ above, not work with the 'hot replacement', so a believe it is a bug.

My rederer main.js imports are:

```
import Vue from 'vue'

import Buefy from 'buefy'
import 'buefy/lib/buefy.css'
import './style/global.css' // <<< Project style file
import 'font-awesome/css/font-awesome.css'
import moment from 'moment'
import 'moment/locale/pt-br'
import injector from 'vue-inject'

```
##### Tell me about your development environment.
  * Node version: 8.2.1
  * NPM version: 5.2.0
  * Operating System: Ubuntu 16.04 LTS

**Comment by SimulatedGREG** [comment]
@AlexVFornazieri 

I am unable to reproduce this myself. You can also try importing your CSS into `App.vue` using CSS `@import`s.

```html
<style>
  @import "~buefy/lib/buefy.css";
  @import "~@/style/global.css";
  @import "~font-awesome/css/font-awesome.css";
</style>
```

**Comment by AlexVFornazieri** [comment]
@SimulatedGREG 

Thanks for reply,

So, I noticed that this is not always happens, so I looked for any pattern. The issue just happens if I run the project in dev and the first file changed and saved is the .css imported via script.

If I run and edit some other file as a .vue first, before editing the css everything works normally. I tried to put the import in App.vue and the same issue occurs.
